mopidy_spotify/__init__.py

```diff
@@ -37,9 +37,9 @@ def get_config_schema(self):
         schema['allow_network'] = config.Boolean()
         schema['allow_playlists'] = config.Boolean()
 
-        schema['search_album_count'] = config.Integer(minimum=0, maximum=200)
-        schema['search_artist_count'] = config.Integer(minimum=0, maximum=200)
-        schema['search_track_count'] = config.Integer(minimum=0, maximum=200)
+        schema['search_album_count'] = config.Integer(minimum=0, maximum=50)
```

Comment:
This will prevent people from starting up if they have set it to anything above 50 explicitly. Would it make sense to leave it at 200, but do a `min(50, ...)` and warn about the downgrade?